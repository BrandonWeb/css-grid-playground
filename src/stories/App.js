import React from 'react';
import { storiesOf } from '@kadira/storybook';
import App from '../App';

const stories = storiesOf('App', module);

stories.add('main app', () => <App />);
