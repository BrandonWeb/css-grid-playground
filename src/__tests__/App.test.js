import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';

it('renders correctly on compatible device', () => {
  const tree = renderer.create(<App isCompatible />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly on incompatible device', () => {
  const tree = renderer.create(<App isCompatible={false} />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly on test runner as incompatible', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
