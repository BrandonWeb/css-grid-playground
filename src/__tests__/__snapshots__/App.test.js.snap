exports[`test renders correctly on compatible device 1`] = `
<div
  className="sc-gzVnrw kugQOZ">
  <div
    className="sc-VigVT jjafDM">
    <h1
      className="sc-htoDjs jVTLrs">
      CSS Grid Playground
    </h1>
    <div
      className="sc-jTzLTM hEuNsT">
      <a
        className="twitter-share-button"
        href="https://twitter.com/intent/tweet">
        Tweet
      </a>
      <a
        aria-label="Star purplecones/css-grid-playground on GitHub"
        className="github-button"
        data-show-count="true"
        href="https://github.com/purplecones/css-grid-playground">
        Star
      </a>
    </div>
  </div>
  <div
    className="sc-gZMcBi dWPVKR">
    <div
      className="sc-gqjmRU itUyyS">
      <div
        className="sc-dnqmqq hFRXBC">
        <h3
          className="sc-iwsKbI kNKUsf">
          Grid Container
        </h3>
      </div>
      <textarea
        className="sc-htpNat XFhiE"
        onChange={[Function]}
        value="display: grid;
      grid-gap: 1rem;
      grid-template-columns: 1fr 5fr 2.5fr;
      grid-template-rows: 1fr 5fr 1fr;
      grid-template-areas:
      \"header header header\"
      \"sidebar main aside\"
      \"sidebar footer footer\";" />
    </div>
    <div
      className="sc-gqjmRU itUyyS">
      <div
        className="sc-dnqmqq hFRXBC">
        <h3
          className="sc-iwsKbI kNKUsf">
          Grid Items
        </h3>
        <div>
          <button
            className="sc-bZQynM dviFfX"
            onClick={[Function]}>
            <img
              alt="Minus icon"
              height="15"
              src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAAAn0lEQVRoQ+2YQQrAIBDE9P+Pbs9CYQ5xKEp63g01kR46x+HPPPz9hwf4u6AFLAANeIWgQLxuAawQAiwABeJ1C2CFEGABKBCvWwArhAALQIF4/coCD9bSBSzSvwp4gG6A9U/KlQXKAvfir/wK7VVUplmgLDjiLRAVlQcsUBYc8RaIisoDFigLjngLREXlAQuUBUe8BaKi8oAFyoIj/vgCLy7LBDFhVD5yAAAAAElFTkSuQmCC"
              width="15" />
          </button>
          <button
            className="sc-bZQynM dviFfX"
            onClick={[Function]}>
            <img
              alt="Plus icon"
              height="15"
              src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAAAnklEQVRoQ+3XQQqAMAwF0Xr/Q+vGjSIS+A1Sea7bNJmMqW5j8WdbPP+hgK872NmB/VZcy1ktQc/EFVDRUwdeKFGIQhUCFAopmUIUolBIgEIhQFOIQg0K3b8iwyOmb79o//QOKGA682vA/3dgFkB/ZBWSLjIXWcWTlzUUohCFQgIUCgGaQhSiUEiAQiFAUygEGG/v7ECcXCWAAiqUOtccVL8oMW7pmEkAAAAASUVORK5CYII="
              width="15" />
          </button>
          <button
            className="sc-bZQynM dviFfX"
            onClick={[Function]}>
            <img
              alt="Clear icon"
              height="15"
              src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAABl0lEQVRoQ+2Y0VEDMQxENx3QCSkBOkg6gMqgA+gASkg6oQRGTDzjudydV9YqcIPv17K9Tyv7PNph499u4/oxAH7bweHAcCCYgX9VQncAHgC8B5PWmn4A8AngqxVo46wDJv4DwB7AM4BXZvGOmCcALwBOAB4ZCAagFl80ZUAU8WUPCoIBMEvfZrKphJiKL9sdWyXLANhiSxsoIEJrswBZECHxnkNcLA1vWJWiZC2PA0oIifgeBxQQMvERgN4zIRUfBfBCyMUrAFiIFPEqgBaEjdvzYPop/iH0W4h51ixleW6uRLzSgdbtVEPIxGcArJWTjUnFD4CZ4mbOgdSFnqfE0oFmxJe5MggVwNo9/+evUeYnxcQwV/VVTNQBjzBPLA0TAegR1DNnFaYXICIkMldSQgoBijV+YLwOyDZWNQo8AErxrbcT/Z9gATLESyAYgM03tqy1aM3W++oKoC2mL/Tr5tn50kxebfIyDpiGGiJD/LScKPHeW2jT7XVHJdw2lC2h26py7DYAHMlKCR0OpKTVsehwwJGslNBvypV0Mel0u1AAAAAASUVORK5CYII="
              width="15" />
          </button>
          <button
            className="sc-bZQynM dviFfX"
            onClick={[Function]}>
            <img
              alt="Hide icon"
              height="15"
              src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAYAAACqaXHeAAAESUlEQVR4Xu2ZjbENQRCFz4uADBABIkAEiAARIAJEgAgQASJABJ4IEAEioD61/Wpe35np2Z+7Vu101dZVdne6+8w53b3zTrRzO9l5/uoAdAbsHIEugZ0ToBfBLoEugZ0j0CWwcwL0LtAl0CWwDgI3JN2UdHm4Lkq65lyfSvop6dtwfZT06djhHUsCJHpb0p0h8Tl5AMQ7Se8HYOasdfDu0gDck3R/gaRLSQLGa0lvlkJhCQCg88MhcXZ+DUMmAPFykM1kn3MBeCLpkSRAqNl3SewegfOL2a+9R43ArFbweylYl5qB/8mMmAoAwb0aClopxi/DLqFfEp9iMIo6gqyuVhZg/QcZUEOfYwFgp0mcoHL2ayhYLyRR1Zc0uga7TZ0pGWADBMxosjEAkDTJ1+gOFdmtYxrar4FA8oAAGKG1AvB8QD9a8Eoj3a1o2iwAW2BNi5aRxdcokGG9x9FzEQDs9ofM0JJbF8374cY/F60HELcaKMxztZpgfsP1agCQDMnnKE/7sWHHnIE2u1izzw0gEfT1YJ2nkuhAZjYkwSxvSAJQszWpBAA6hvY+eYoc99AXbYwR1wwnvrWlwVDAWLPF0DBaLxldiM0xY2Tm/6hTvHfBvVisCzkASJBi5w2K48Ba2m/3QCSnlt23JSMWsDE/Cv5hJhuUk8gBsD7oUvJQjHtpexkLgH8+YkIEaM0/AMEEvke8nQMhdVJKvtTatgyAJV1qmWcgGAAUPCjqrdbXYUOqtagFtlZuYog6io+X2lSaT0ogUGhPDQBfVFIEQStnWyyCuTipZ7nh7BwAvFiSAAjmQPDIPpNEe6pZCwui3Wd93wZLTC0lfyABC7oGAn0+LYK+rUWVGx/QFOaUhhiSh43RLO87iq/u+KHl5na+WAQjEPxUlRtJozqQ+gBAA4LEGaJqvd/ejfxy/21h4ArboDmpDRR33Td9Ogz9i48hG4KIHfaQfG2AOyfRaBSGrn6qMg2i+Zxk/haXqMlPvJ/rVrarjMa5GkSHAJhRo7DFV9MsC9r8n2qaSREQIh2PxYBY0H567IZ0rBblPsTCmhJNWxYk+sx9aHAfILxzRlGksqRBbX8Qk/NtPvlgA5yqtQJg+qJIRed05jD8FI2CG+5Hn9B+Gc4fkWbtw+zsnTEAWBuDDbUTmTQgk0lTMBlA0C7tLDpnsFcpwux6s/zGAmCOCAwgWg4leAfmUDRbD0fROUWt9XjNasFooKcCYEAQIJW3VRYEaBdB205Bc8AEWLtaFALd8d8yP2TXmwvAVCBakqs9MztxW3wpAGw9O8PPfYfPTZr3OZdgt5tOfFscLg2A+YTSgME1FwySJmGu5uLWkjzPHAsA758qbn/y4t+m+fQ5qwl0DvsT2rEmysltsBXY/+a5tRiwWUA6AJvdmpUC6wxYCejNuukM2OzWrBRYZ8BKQG/WTWfAZrdmpcA6A1YCerNuOgM2uzUrBfYH/6zsQTLY9P4AAAAASUVORK5CYII="
              width="15" />
          </button>
        </div>
      </div>
      <textarea
        className="sc-htpNat XFhiE"
        onChange={[Function]}
        value="background: lightsalmon;
      border: 2px solid black;
      " />
    </div>
  </div>
  <div
    className="sc-bdVaJa NFzwl">
    <div
      className="sc-bwzfXH iqsQaz">
      <div
        className="sc-bxivhb ldFVBS">
        <textarea
          className="sc-htpNat XFhiE"
          onChange={[Function]}
          value="grid-area: header;" />
        <div
          className="sc-ifAKCX bfPppq">
          <div
            className="sc-EHOje imGZnt">
            1
          </div>
        </div>
      </div>
      <div
        className="sc-bxivhb gqDXHQ">
        <textarea
          className="sc-htpNat XFhiE"
          onChange={[Function]}
          value="grid-area: sidebar;" />
        <div
          className="sc-ifAKCX bfPppq">
          <div
            className="sc-EHOje imGZnt">
            2
          </div>
        </div>
      </div>
      <div
        className="sc-bxivhb hczrtq">
        <textarea
          className="sc-htpNat XFhiE"
          onChange={[Function]}
          value="grid-area: main;" />
        <div
          className="sc-ifAKCX bfPppq">
          <div
            className="sc-EHOje imGZnt">
            3
          </div>
        </div>
      </div>
      <div
        className="sc-bxivhb kxEueQ">
        <textarea
          className="sc-htpNat XFhiE"
          onChange={[Function]}
          value="grid-area: aside;" />
        <div
          className="sc-ifAKCX bfPppq">
          <div
            className="sc-EHOje imGZnt">
            4
          </div>
        </div>
      </div>
      <div
        className="sc-bxivhb kuMbkU">
        <textarea
          className="sc-htpNat XFhiE"
          onChange={[Function]}
          value="grid-area: footer;" />
        <div
          className="sc-ifAKCX bfPppq">
          <div
            className="sc-EHOje imGZnt">
            5
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`test renders correctly on incompatible device 1`] = `
<div
  className="sc-gzVnrw kugQOZ">
  <div
    className="sc-VigVT jjafDM">
    <h1
      className="sc-htoDjs jVTLrs">
      CSS Grid Playground
    </h1>
    <div
      className="sc-jTzLTM hEuNsT">
      <a
        className="twitter-share-button"
        href="https://twitter.com/intent/tweet">
        Tweet
      </a>
      <a
        aria-label="Star purplecones/css-grid-playground on GitHub"
        className="github-button"
        data-show-count="true"
        href="https://github.com/purplecones/css-grid-playground">
        Star
      </a>
    </div>
  </div>
  <div
    className="sc-gZMcBi dWPVKR">
    <div
      className="sc-gqjmRU itUyyS">
      <div
        className="sc-dnqmqq hFRXBC">
        <h3
          className="sc-iwsKbI kNKUsf">
          Grid Container
        </h3>
      </div>
      <textarea
        className="sc-htpNat XFhiE"
        onChange={[Function]}
        value="display: grid;
      grid-gap: 1rem;
      grid-template-columns: 1fr 5fr 2.5fr;
      grid-template-rows: 1fr 5fr 1fr;
      grid-template-areas:
      \"header header header\"
      \"sidebar main aside\"
      \"sidebar footer footer\";" />
    </div>
    <div
      className="sc-gqjmRU itUyyS">
      <div
        className="sc-dnqmqq hFRXBC">
        <h3
          className="sc-iwsKbI kNKUsf">
          Grid Items
        </h3>
        <div>
          <button
            className="sc-bZQynM dviFfX"
            onClick={[Function]}>
            <img
              alt="Minus icon"
              height="15"
              src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAAAn0lEQVRoQ+2YQQrAIBDE9P+Pbs9CYQ5xKEp63g01kR46x+HPPPz9hwf4u6AFLAANeIWgQLxuAawQAiwABeJ1C2CFEGABKBCvWwArhAALQIF4/coCD9bSBSzSvwp4gG6A9U/KlQXKAvfir/wK7VVUplmgLDjiLRAVlQcsUBYc8RaIisoDFigLjngLREXlAQuUBUe8BaKi8oAFyoIj/vgCLy7LBDFhVD5yAAAAAElFTkSuQmCC"
              width="15" />
          </button>
          <button
            className="sc-bZQynM dviFfX"
            onClick={[Function]}>
            <img
              alt="Plus icon"
              height="15"
              src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAAAnklEQVRoQ+3XQQqAMAwF0Xr/Q+vGjSIS+A1Sea7bNJmMqW5j8WdbPP+hgK872NmB/VZcy1ktQc/EFVDRUwdeKFGIQhUCFAopmUIUolBIgEIhQFOIQg0K3b8iwyOmb79o//QOKGA682vA/3dgFkB/ZBWSLjIXWcWTlzUUohCFQgIUCgGaQhSiUEiAQiFAUygEGG/v7ECcXCWAAiqUOtccVL8oMW7pmEkAAAAASUVORK5CYII="
              width="15" />
          </button>
          <button
            className="sc-bZQynM dviFfX"
            onClick={[Function]}>
            <img
              alt="Clear icon"
              height="15"
              src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAABl0lEQVRoQ+2Y0VEDMQxENx3QCSkBOkg6gMqgA+gASkg6oQRGTDzjudydV9YqcIPv17K9Tyv7PNph499u4/oxAH7bweHAcCCYgX9VQncAHgC8B5PWmn4A8AngqxVo46wDJv4DwB7AM4BXZvGOmCcALwBOAB4ZCAagFl80ZUAU8WUPCoIBMEvfZrKphJiKL9sdWyXLANhiSxsoIEJrswBZECHxnkNcLA1vWJWiZC2PA0oIifgeBxQQMvERgN4zIRUfBfBCyMUrAFiIFPEqgBaEjdvzYPop/iH0W4h51ixleW6uRLzSgdbtVEPIxGcArJWTjUnFD4CZ4mbOgdSFnqfE0oFmxJe5MggVwNo9/+evUeYnxcQwV/VVTNQBjzBPLA0TAegR1DNnFaYXICIkMldSQgoBijV+YLwOyDZWNQo8AErxrbcT/Z9gATLESyAYgM03tqy1aM3W++oKoC2mL/Tr5tn50kxebfIyDpiGGiJD/LScKPHeW2jT7XVHJdw2lC2h26py7DYAHMlKCR0OpKTVsehwwJGslNBvypV0Mel0u1AAAAAASUVORK5CYII="
              width="15" />
          </button>
          <button
            className="sc-bZQynM dviFfX"
            onClick={[Function]}>
            <img
              alt="Hide icon"
              height="15"
              src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAYAAACqaXHeAAAESUlEQVR4Xu2ZjbENQRCFz4uADBABIkAEiAARIAJEgAgQASJABJ4IEAEioD61/Wpe35np2Z+7Vu101dZVdne6+8w53b3zTrRzO9l5/uoAdAbsHIEugZ0ToBfBLoEugZ0j0CWwcwL0LtAl0CWwDgI3JN2UdHm4Lkq65lyfSvop6dtwfZT06djhHUsCJHpb0p0h8Tl5AMQ7Se8HYOasdfDu0gDck3R/gaRLSQLGa0lvlkJhCQCg88MhcXZ+DUMmAPFykM1kn3MBeCLpkSRAqNl3SewegfOL2a+9R43ArFbweylYl5qB/8mMmAoAwb0aClopxi/DLqFfEp9iMIo6gqyuVhZg/QcZUEOfYwFgp0mcoHL2ayhYLyRR1Zc0uga7TZ0pGWADBMxosjEAkDTJ1+gOFdmtYxrar4FA8oAAGKG1AvB8QD9a8Eoj3a1o2iwAW2BNi5aRxdcokGG9x9FzEQDs9ofM0JJbF8374cY/F60HELcaKMxztZpgfsP1agCQDMnnKE/7sWHHnIE2u1izzw0gEfT1YJ2nkuhAZjYkwSxvSAJQszWpBAA6hvY+eYoc99AXbYwR1wwnvrWlwVDAWLPF0DBaLxldiM0xY2Tm/6hTvHfBvVisCzkASJBi5w2K48Ba2m/3QCSnlt23JSMWsDE/Cv5hJhuUk8gBsD7oUvJQjHtpexkLgH8+YkIEaM0/AMEEvke8nQMhdVJKvtTatgyAJV1qmWcgGAAUPCjqrdbXYUOqtagFtlZuYog6io+X2lSaT0ogUGhPDQBfVFIEQStnWyyCuTipZ7nh7BwAvFiSAAjmQPDIPpNEe6pZCwui3Wd93wZLTC0lfyABC7oGAn0+LYK+rUWVGx/QFOaUhhiSh43RLO87iq/u+KHl5na+WAQjEPxUlRtJozqQ+gBAA4LEGaJqvd/ejfxy/21h4ArboDmpDRR33Td9Ogz9i48hG4KIHfaQfG2AOyfRaBSGrn6qMg2i+Zxk/haXqMlPvJ/rVrarjMa5GkSHAJhRo7DFV9MsC9r8n2qaSREQIh2PxYBY0H567IZ0rBblPsTCmhJNWxYk+sx9aHAfILxzRlGksqRBbX8Qk/NtPvlgA5yqtQJg+qJIRed05jD8FI2CG+5Hn9B+Gc4fkWbtw+zsnTEAWBuDDbUTmTQgk0lTMBlA0C7tLDpnsFcpwux6s/zGAmCOCAwgWg4leAfmUDRbD0fROUWt9XjNasFooKcCYEAQIJW3VRYEaBdB205Bc8AEWLtaFALd8d8yP2TXmwvAVCBakqs9MztxW3wpAGw9O8PPfYfPTZr3OZdgt5tOfFscLg2A+YTSgME1FwySJmGu5uLWkjzPHAsA758qbn/y4t+m+fQ5qwl0DvsT2rEmysltsBXY/+a5tRiwWUA6AJvdmpUC6wxYCejNuukM2OzWrBRYZ8BKQG/WTWfAZrdmpcA6A1YCerNuOgM2uzUrBfYH/6zsQTLY9P4AAAAASUVORK5CYII="
              width="15" />
          </button>
        </div>
      </div>
      <textarea
        className="sc-htpNat XFhiE"
        onChange={[Function]}
        value="background: lightsalmon;
      border: 2px solid black;
      " />
    </div>
  </div>
  <div
    className="sc-bdVaJa NFzwl">
    <div
      className="sc-bwzfXH iqsQaz">
      <div
        className="sc-bxivhb ldFVBS">
        <textarea
          className="sc-htpNat XFhiE"
          onChange={[Function]}
          value="grid-area: header;" />
        <div
          className="sc-ifAKCX bfPppq">
          <div
            className="sc-EHOje imGZnt">
            1
          </div>
        </div>
      </div>
      <div
        className="sc-bxivhb gqDXHQ">
        <textarea
          className="sc-htpNat XFhiE"
          onChange={[Function]}
          value="grid-area: sidebar;" />
        <div
          className="sc-ifAKCX bfPppq">
          <div
            className="sc-EHOje imGZnt">
            2
          </div>
        </div>
      </div>
      <div
        className="sc-bxivhb hczrtq">
        <textarea
          className="sc-htpNat XFhiE"
          onChange={[Function]}
          value="grid-area: main;" />
        <div
          className="sc-ifAKCX bfPppq">
          <div
            className="sc-EHOje imGZnt">
            3
          </div>
        </div>
      </div>
      <div
        className="sc-bxivhb kxEueQ">
        <textarea
          className="sc-htpNat XFhiE"
          onChange={[Function]}
          value="grid-area: aside;" />
        <div
          className="sc-ifAKCX bfPppq">
          <div
            className="sc-EHOje imGZnt">
            4
          </div>
        </div>
      </div>
      <div
        className="sc-bxivhb kuMbkU">
        <textarea
          className="sc-htpNat XFhiE"
          onChange={[Function]}
          value="grid-area: footer;" />
        <div
          className="sc-ifAKCX bfPppq">
          <div
            className="sc-EHOje imGZnt">
            5
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`test renders correctly on test runner as incompatible 1`] = `
<div
  className="sc-gzVnrw kugQOZ">
  <div
    className="sc-VigVT jjafDM">
    <h1
      className="sc-htoDjs jVTLrs">
      CSS Grid Playground
    </h1>
    <div
      className="sc-jTzLTM hEuNsT">
      <a
        className="twitter-share-button"
        href="https://twitter.com/intent/tweet">
        Tweet
      </a>
      <a
        aria-label="Star purplecones/css-grid-playground on GitHub"
        className="github-button"
        data-show-count="true"
        href="https://github.com/purplecones/css-grid-playground">
        Star
      </a>
    </div>
  </div>
  <div
    className="sc-gZMcBi dWPVKR">
    <div
      className="sc-gqjmRU itUyyS">
      <div
        className="sc-dnqmqq hFRXBC">
        <h3
          className="sc-iwsKbI kNKUsf">
          Grid Container
        </h3>
      </div>
      <textarea
        className="sc-htpNat XFhiE"
        onChange={[Function]}
        value="display: grid;
      grid-gap: 1rem;
      grid-template-columns: 1fr 5fr 2.5fr;
      grid-template-rows: 1fr 5fr 1fr;
      grid-template-areas:
      \"header header header\"
      \"sidebar main aside\"
      \"sidebar footer footer\";" />
    </div>
    <div
      className="sc-gqjmRU itUyyS">
      <div
        className="sc-dnqmqq hFRXBC">
        <h3
          className="sc-iwsKbI kNKUsf">
          Grid Items
        </h3>
        <div>
          <button
            className="sc-bZQynM dviFfX"
            onClick={[Function]}>
            <img
              alt="Minus icon"
              height="15"
              src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAAAn0lEQVRoQ+2YQQrAIBDE9P+Pbs9CYQ5xKEp63g01kR46x+HPPPz9hwf4u6AFLAANeIWgQLxuAawQAiwABeJ1C2CFEGABKBCvWwArhAALQIF4/coCD9bSBSzSvwp4gG6A9U/KlQXKAvfir/wK7VVUplmgLDjiLRAVlQcsUBYc8RaIisoDFigLjngLREXlAQuUBUe8BaKi8oAFyoIj/vgCLy7LBDFhVD5yAAAAAElFTkSuQmCC"
              width="15" />
          </button>
          <button
            className="sc-bZQynM dviFfX"
            onClick={[Function]}>
            <img
              alt="Plus icon"
              height="15"
              src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAAAnklEQVRoQ+3XQQqAMAwF0Xr/Q+vGjSIS+A1Sea7bNJmMqW5j8WdbPP+hgK872NmB/VZcy1ktQc/EFVDRUwdeKFGIQhUCFAopmUIUolBIgEIhQFOIQg0K3b8iwyOmb79o//QOKGA682vA/3dgFkB/ZBWSLjIXWcWTlzUUohCFQgIUCgGaQhSiUEiAQiFAUygEGG/v7ECcXCWAAiqUOtccVL8oMW7pmEkAAAAASUVORK5CYII="
              width="15" />
          </button>
          <button
            className="sc-bZQynM dviFfX"
            onClick={[Function]}>
            <img
              alt="Clear icon"
              height="15"
              src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAABl0lEQVRoQ+2Y0VEDMQxENx3QCSkBOkg6gMqgA+gASkg6oQRGTDzjudydV9YqcIPv17K9Tyv7PNph499u4/oxAH7bweHAcCCYgX9VQncAHgC8B5PWmn4A8AngqxVo46wDJv4DwB7AM4BXZvGOmCcALwBOAB4ZCAagFl80ZUAU8WUPCoIBMEvfZrKphJiKL9sdWyXLANhiSxsoIEJrswBZECHxnkNcLA1vWJWiZC2PA0oIifgeBxQQMvERgN4zIRUfBfBCyMUrAFiIFPEqgBaEjdvzYPop/iH0W4h51ixleW6uRLzSgdbtVEPIxGcArJWTjUnFD4CZ4mbOgdSFnqfE0oFmxJe5MggVwNo9/+evUeYnxcQwV/VVTNQBjzBPLA0TAegR1DNnFaYXICIkMldSQgoBijV+YLwOyDZWNQo8AErxrbcT/Z9gATLESyAYgM03tqy1aM3W++oKoC2mL/Tr5tn50kxebfIyDpiGGiJD/LScKPHeW2jT7XVHJdw2lC2h26py7DYAHMlKCR0OpKTVsehwwJGslNBvypV0Mel0u1AAAAAASUVORK5CYII="
              width="15" />
          </button>
          <button
            className="sc-bZQynM dviFfX"
            onClick={[Function]}>
            <img
              alt="Hide icon"
              height="15"
              src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAYAAACqaXHeAAAESUlEQVR4Xu2ZjbENQRCFz4uADBABIkAEiAARIAJEgAgQASJABJ4IEAEioD61/Wpe35np2Z+7Vu101dZVdne6+8w53b3zTrRzO9l5/uoAdAbsHIEugZ0ToBfBLoEugZ0j0CWwcwL0LtAl0CWwDgI3JN2UdHm4Lkq65lyfSvop6dtwfZT06djhHUsCJHpb0p0h8Tl5AMQ7Se8HYOasdfDu0gDck3R/gaRLSQLGa0lvlkJhCQCg88MhcXZ+DUMmAPFykM1kn3MBeCLpkSRAqNl3SewegfOL2a+9R43ArFbweylYl5qB/8mMmAoAwb0aClopxi/DLqFfEp9iMIo6gqyuVhZg/QcZUEOfYwFgp0mcoHL2ayhYLyRR1Zc0uga7TZ0pGWADBMxosjEAkDTJ1+gOFdmtYxrar4FA8oAAGKG1AvB8QD9a8Eoj3a1o2iwAW2BNi5aRxdcokGG9x9FzEQDs9ofM0JJbF8374cY/F60HELcaKMxztZpgfsP1agCQDMnnKE/7sWHHnIE2u1izzw0gEfT1YJ2nkuhAZjYkwSxvSAJQszWpBAA6hvY+eYoc99AXbYwR1wwnvrWlwVDAWLPF0DBaLxldiM0xY2Tm/6hTvHfBvVisCzkASJBi5w2K48Ba2m/3QCSnlt23JSMWsDE/Cv5hJhuUk8gBsD7oUvJQjHtpexkLgH8+YkIEaM0/AMEEvke8nQMhdVJKvtTatgyAJV1qmWcgGAAUPCjqrdbXYUOqtagFtlZuYog6io+X2lSaT0ogUGhPDQBfVFIEQStnWyyCuTipZ7nh7BwAvFiSAAjmQPDIPpNEe6pZCwui3Wd93wZLTC0lfyABC7oGAn0+LYK+rUWVGx/QFOaUhhiSh43RLO87iq/u+KHl5na+WAQjEPxUlRtJozqQ+gBAA4LEGaJqvd/ejfxy/21h4ArboDmpDRR33Td9Ogz9i48hG4KIHfaQfG2AOyfRaBSGrn6qMg2i+Zxk/haXqMlPvJ/rVrarjMa5GkSHAJhRo7DFV9MsC9r8n2qaSREQIh2PxYBY0H567IZ0rBblPsTCmhJNWxYk+sx9aHAfILxzRlGksqRBbX8Qk/NtPvlgA5yqtQJg+qJIRed05jD8FI2CG+5Hn9B+Gc4fkWbtw+zsnTEAWBuDDbUTmTQgk0lTMBlA0C7tLDpnsFcpwux6s/zGAmCOCAwgWg4leAfmUDRbD0fROUWt9XjNasFooKcCYEAQIJW3VRYEaBdB205Bc8AEWLtaFALd8d8yP2TXmwvAVCBakqs9MztxW3wpAGw9O8PPfYfPTZr3OZdgt5tOfFscLg2A+YTSgME1FwySJmGu5uLWkjzPHAsA758qbn/y4t+m+fQ5qwl0DvsT2rEmysltsBXY/+a5tRiwWUA6AJvdmpUC6wxYCejNuukM2OzWrBRYZ8BKQG/WTWfAZrdmpcA6A1YCerNuOgM2uzUrBfYH/6zsQTLY9P4AAAAASUVORK5CYII="
              width="15" />
          </button>
        </div>
      </div>
      <textarea
        className="sc-htpNat XFhiE"
        onChange={[Function]}
        value="background: lightsalmon;
      border: 2px solid black;
      " />
    </div>
  </div>
  <div
    className="sc-bdVaJa NFzwl">
    <div
      className="sc-bwzfXH iqsQaz">
      <div
        className="sc-bxivhb ldFVBS">
        <textarea
          className="sc-htpNat XFhiE"
          onChange={[Function]}
          value="grid-area: header;" />
        <div
          className="sc-ifAKCX bfPppq">
          <div
            className="sc-EHOje imGZnt">
            1
          </div>
        </div>
      </div>
      <div
        className="sc-bxivhb gqDXHQ">
        <textarea
          className="sc-htpNat XFhiE"
          onChange={[Function]}
          value="grid-area: sidebar;" />
        <div
          className="sc-ifAKCX bfPppq">
          <div
            className="sc-EHOje imGZnt">
            2
          </div>
        </div>
      </div>
      <div
        className="sc-bxivhb hczrtq">
        <textarea
          className="sc-htpNat XFhiE"
          onChange={[Function]}
          value="grid-area: main;" />
        <div
          className="sc-ifAKCX bfPppq">
          <div
            className="sc-EHOje imGZnt">
            3
          </div>
        </div>
      </div>
      <div
        className="sc-bxivhb kxEueQ">
        <textarea
          className="sc-htpNat XFhiE"
          onChange={[Function]}
          value="grid-area: aside;" />
        <div
          className="sc-ifAKCX bfPppq">
          <div
            className="sc-EHOje imGZnt">
            4
          </div>
        </div>
      </div>
      <div
        className="sc-bxivhb kuMbkU">
        <textarea
          className="sc-htpNat XFhiE"
          onChange={[Function]}
          value="grid-area: footer;" />
        <div
          className="sc-ifAKCX bfPppq">
          <div
            className="sc-EHOje imGZnt">
            5
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
